### PR TITLE
Treat race none more correctly

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/models/grda_warehouse/hud/client.rb [hunk 1]
```diff
@@ -2065,6 +2089,16 @@ def race_string(destination_id:, scope_limit: self.class.destination)
       return 'NativeHIPacific' if @race_native_hi_other_pacific.include?(destination_id)
       return 'White' if @race_white.include?(destination_id)
 
+      if include_none_reason
+        @doesnt_know ||= @limited_scope.where(id: self.class.race_doesnt_know.select(:id)).distinct.pluck(:id).to_set
+        @refused ||= @limited_scope.where(id: self.class.race_refused.select(:id)).distinct.pluck(:id).to_set
+        @none ||= @limited_scope.where(id: self.class.race_none.select(:id)).distinct.pluck(:id).to_set
```
Comment (awisema) [inline app/models/grda_warehouse/hud/client.rb]: I think this is not used?

**Review by awisema — CHANGES_REQUESTED** [review]

File: app/models/grda_warehouse/hud/client.rb (hunk 1 above)
Comment (eanders) [reply app/models/grda_warehouse/hud/client.rb]: That's used here: https://github.com/greenriver/hmis-warehouse/pull/2078/files#diff-4e014a29a6b3c20e59f3a3a2163533328e94f9277aad680ef1c92129f7c91afeR75

Comment (eanders) [reply app/models/grda_warehouse/hud/client.rb]: Oh, @none is not used.

Comment (eanders) [reply app/models/grda_warehouse/hud/client.rb]: I think you are correct.

**Pull request merged**